2021q2/FreshPorts.adoc

```diff
@@ -0,0 +1,57 @@
+== FreshPorts
+
+Link:	link:https://freshports.org/[FreshPorts]
+Link:	link:https://news.freshports.org/[FreshPorts blog]
```

Comment:
In the asciidoc in the tree, there is no space after the colon

Reply:
I sent you a pull-request with these corrected.

Reply:
Thank you.